index.html

```diff
@@ -23,7 +23,11 @@
 
 	<!-- to-do main container -->
 	<section class="todo-main-container">
-		<div id="newList"></div>
+		<div
+		class="progress-box"
+		id="newList">
+			<h3>New To Dos</h3>
+		</div>
```

Comment:
Added to this div so that it would be more descriptive for students and so that it would style correctly.